### PR TITLE
PIMP-73 change the 3D cubes to rotate on a timer rather than a hover …

### PR DESCRIPTION
**Pull request by rimpey**
https://womenhackfornonprofits.atlassian.net/browse/PIMP-73

I've changed the code in the /scripts/scripts.js and the /styles/components/_cubes.scss so that now the cubes are activated using a timer. 

@lili2311 @tanyapowell

### CODE REVIEW

**Comment by lili2311** [comment]
@rimpey could you post a little gif of what it looks like now?

**Comment by lili2311** [comment]
Also any chance we can do this with just css transitions no js? Ideally want try to avoid DOM operations especially on the homepage.

**Comment by lili2311** [comment]
Example: http://cssdeck.com/labs/css3-flipping-cube

**Comment by lili2311** [comment]
@rimpey some changes to the behaviour:
- [ ] On mouse over they stop flipping
- [ ] if no mouse over they flip on a timed basis
- [ ] all tiles are a clickable link as the whole tile and take you to the relevant page.

**Comment by rimpey** [comment]
Hello @lili2311 

As mentioned on slack this is a link to a demonstration of the cubes:
http://womenhackfornonprofits.github.io/pimpmycause-rebuild/ 

Just to reiterate I did use css3 transitions and javascript to take care of the timer event.

@lili2311 It seems you have listed change requests here in Github. I am of course keen to do this but for the purposes of this 'PIMP-73' Jira task the specified requirements have been completed so the PR may be merged. Otherwise it seems to defeat the purpose of having sprints and tasks per sprint if you just keep an old task open and keep adding change requests. I would consider it to make sense to establish small tasks, complete them and then draw up other small tasks for the next sprint. This is just a very simple bit of code. 

@tanyapowell you accepted Lilli's PR however just a small note for next time; make sure you check the .png of the front page design I've attached to the Jira task PIMP-71 as the registration panel and contact panel that Lili completed do not match the design from Jeanette; incorrect button colour no hover event and alignment not as per design. Just a small note for next time.

Thanks
Rhonda

**Comment by lili2311** [comment]
@rimpey the PR was initial commit, it can be tweaked over time to look closer and closer to the actual design.

**Comment by rimpey** [comment]
Okay it just needs to have a better description in the Jira task to make this clear. Thanks.

**Comment by tanyapowell** [comment]
@rimpey which PR are you referring to that I accepted? Would you mind adding the link to it it here to refresh my memory

**Comment by rimpey** [comment]
Hi @tanyapowell  just bringing your attention to .png on PIMP-71 to compare front end dev work against: https://womenhackfornonprofits.atlassian.net/browse/PIMP-71

https://github.com/womenhackfornonprofits/pimpmycause-rebuild/pull/6
https://github.com/womenhackfornonprofits/pimpmycause-rebuild/pull/7

File: frontend/src/scripts/scripts.js [hunk 1]
```diff
@@ -1,5 +1,27 @@
 $(document).foundation();
 
-$(document).ready(function() {
+$(window).on("load", function() {
+    
+    // rotating the cubes on the front hero panel using a timer
+    
+    var x = true;
```
Comment (tanyapowell) [inline frontend/src/scripts/scripts.js]: Could `var x` be renamed to a more meaningful name? Makes it easier for someone else to come along and read your code once the project has been handed over :smile:

**Comment by rimpey** [comment]
Hello @tanyapowell is that better :) Thanks :) Changed 'x' to 'loop_condition' much better to read I quite agree :)

**Comment by lili2311** [comment]
@rimpey  please create a Jira ticket  and post a link here to have the other functionality added:

 Done:
- [x] if  no mouse over they flip on a timed basis 

To add:
- [ ] on mouse over they stop flipping
- [ ] all tiles are a clickable link as the whole tile and take you to the relevant page.

**Pull request merged**